google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineStandardRunTask.java

```diff
@@ -51,12 +53,19 @@ public AppEngineStandardRunTask(@NotNull RunConfiguration runConfig, @Nullable S
   public void execute(ProcessStartListener startListener) {
     CloudSdkService sdkService = CloudSdkService.getInstance();
 
-    CloudSdk.Builder sdkBuilder = new CloudSdk.Builder()
+    CloudSdk sdk = new CloudSdk.Builder()
         .sdkPath(sdkService.getSdkHomePath())
         .async(true)
-        .startListener(startListener);
+        .startListener(startListener)
+        .build();
 
-    CloudSdkAppEngineDevServer devServer = new CloudSdkAppEngineDevServer(sdkBuilder.build());
+    try {
+      sdk.validateAppEngineJavaComponents();
+    } catch (AppEngineJavaComponentsNotInstalledException ex) {
+      return;
```

Comment:
This method actually doesn't return anything. AppEngineExecutor.getProcess() would. I could try and add some Javadoc there?

I feel like the right way to do this with exceptions would be to throw a checked one, which conflicts with the declaration of `execute()`.